backend/testfiles/execution/stdlib/parser.dark

```diff
@@ -77,6 +77,12 @@ module ParseToSimplifiedTree =
                               children = [] } ] } ] } ] }
 
 
+  ("" |> Builtin.parserParseToSimplifiedTree) = ParsedNode
```

Comment:
Maybe this test isn't necessary?

Reply:
No, I think it's good to have a "what happens if we try to parse an empty string" test -- I see you had to adjust code to deal with the case, I think that warrants a test.